### PR TITLE
Issue #SB-7884 feat: Form data migration Documentation

### PR DESCRIPTION
**Pull request by anoophm**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://253.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://253.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/form_data_migration/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://253.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://253.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/form_data_migration/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/configuring_sunbird/form_data_migration.md [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
+
+
+## Migration Script
+
+To migrate form data:
+
+1. Checkout release 1.11 sunbird portal from Github.
+
+2. Move to ```src/app/cassandra_migration/release-1.11/form-plugin-migration```
+
```
Comment (Sunitashelar) [inline developer-docs/configuring_sunbird/form_data_migration.md]: @anoophm what do you mean by 'Move to'? do you mean to go to the folder mentioned in the path given ?

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://253.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://253.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/form_data_migration/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/configuring_sunbird/form_data_migration.md (hunk 1 above)
Comment (anoophm) [reply developer-docs/configuring_sunbird/form_data_migration.md]: yes go to that folder

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://253.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://253.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/form_data_migration/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**